tests/cnf/core/network/metallb/internal/frr/frr.go

```diff
@@ -378,6 +414,10 @@ func getBgpStatus(frrPod *pod.Builder, cmd string) (*bgpStatus, error) {
 		return nil, err
 	}
 
+	if len(bgpStatus.Routes) == 0 {
```

Comment:
Why do we want to check routes in the func with name getBgpStatus? Please make this validation separately

Reply:
Duplicate code I have what I need in ValidateLocalPref. 
Removed